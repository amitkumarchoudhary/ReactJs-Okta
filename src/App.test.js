import React from 'react';
// import ReactDOM from 'react-dom';
import Demo from './Demo';
import Demo2 from './Demo2';

import { create } from 'react-test-renderer'

describe('My first snapshot test demo 1',()=>{
    test('testing app button demo', () => {
    let tree = create(<Demo />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})
describe('My first snapshot test demo2',()=>{
    test('testing app button', () => {
    let tree = create(<Demo2 />)
    expect(tree.toJSON()).toMatchSnapshot();
  })
})

// it('renders without crashing', () => {
//   const div = document.createElement('div');
//   ReactDOM.render(<App />, div);
//   ReactDOM.unmountComponentAtNode(div);
// });
