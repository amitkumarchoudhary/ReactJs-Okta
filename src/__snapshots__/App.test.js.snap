// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My first snapshot test demo 1 testing app button demo 1`] = `
<div>
  <button>
    Show 1
  </button>
</div>
`;

exports[`My first snapshot test demo2 testing app button 1`] = `
<div>
  <button>
    Show 2
  </button>
</div>
`;
